modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1951,7 +1951,7 @@ void setViewportLength(double value) {
     /**
      * Gets the breadth of a specific cell
      */
-    double getCellBreadth(Cell cell) {
+    double getCellBreadth(T cell) {
```

Comment:
is there a possibility of introducing a runtime errors?

Reply:
No, this is just cosmetic and more correct as every cell inside the VirtualFlow is bound to the type parameter T.

Reply:
this method should have been private.  
I see no other callers outside of VirtualFlow, so it's probably safe.